omisego-sdk/src/main/java/co/omisego/omisego/websocket/listener/SocketCustomEventListener.kt

```diff
@@ -0,0 +1,120 @@
+package co.omisego.omisego.websocket.listener
+
+/*
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 8/5/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+import co.omisego.omisego.model.APIError
+import co.omisego.omisego.model.socket.SocketReceive
+import co.omisego.omisego.model.transaction.consumption.TransactionConsumption
+import co.omisego.omisego.websocket.event.SocketEvent
+import co.omisego.omisego.websocket.event.TransactionConsumptionFinalizedEvent
+import co.omisego.omisego.websocket.event.TransactionConsumptionRequestEvent
+import co.omisego.omisego.websocket.strategy.FilterStrategy
+
+interface SocketCustomEventListener {
+    var strategy: FilterStrategy
```

Comment:
This should not be here, only in the `SimpleSocketCustomEventListener` implementation, since it's the particularity of it.